src/utils/self_test/self_test.c

```diff
@@ -1686,10 +1716,11 @@ int main(int argc, char *argv[])
 			{"randomize-endpoints", no_argument, 0, 'q'},
 			{"path", required_argument, 0, 'p'},
 			{"nopmix", no_argument, 0, 'n'},
+			{"use-daos-agent-env", no_argument, 0, 'u'},
 			{0, 0, 0, 0}
 		};
 
-		c = getopt_long(argc, argv, "g:m:e:s:r:i:a:btnqp:",
+		c = getopt_long(argc, argv, "g:m:e:s:r:i:a:btnqp:u:",
```

Comment:
should this not be just  'u' without ':'? ':' usually means there needs to be parameter after -u

Reply:
Correct.  `-u` takes no arguments.  fb4a83b4bb1b043ba7ea22d0fb0839a480c274e1